### DISCUSSION TITLE
Please tell me how to delete a user

### DISCUSSION THREAD

**Discussion by kzkondo** [Budibase/budibase#9721] [Help] [answered]
**Checklist**
- [✔️] I have searched budibase discussions and github issues to check if my issue already exists

**Hosting**
- Self
  - Method: docker single image
  -   - Budibase Version: 2.3.15

**Describe**
Please tell me how to delete a user

**To Reproduce**
1. /builder/portal/users/users   Remove a user from
2. Deleted users exist in the app users.
3. I don't know how to delete a user as I don't have the ability to delete a user from the app's user table.

**Expected behavior**
This user is a Google account logged in with Authentication.

**Screenshots**
![スクリーンショット 2023-02-17 14 45 30](https://user-images.githubusercontent.com/93172658/219559916-071cff51-c471-4822-a522-5611cfe08f65.png)
![名称未設定-1](https://user-images.githubusercontent.com/93172658/219559936-347a72d5-c3ed-410a-8746-bf189129a5fc.png)

**Desktop (please complete the following information):**
 - OS: MacOS 12.6.3
 - Browser chrome, safari
 - Version latest

**Comment by melohagan** [comment, marked as answer]
Hey @kzkondo 

You cannot delete users from the users' table, however as you can see the **Status** field has set the user to **Inactive**. This indicates that the user no longer has access to the app.

The reason we don't delete or allows users to be deleted from the users' table is because it could affect auditing. For example you have the *Created-By* field.

Even if a user had access revoked from an app (or was deleted), you would still want to see who had created previous records.

**Reply by kzkondo** [reply]
Hey @melohagan 
thank you!
Is it possible to remove it in the future?
I have a user just to test,
After testing, you won't need it.
Occasionally, Automation incorrectly sends emails to inactive user email addresses.

**Reply by melohagan** [reply]
I believe if you export the app and re-import it then the users won't be copied in the new app - you could try that!
https://docs.budibase.com/docs/export-and-import-apps

**Reply by kzkondo** [reply]
Hi! @melohagan 
Thank you for thinking of a workaround.
